src/Uri/JHtmlSidebar.php

```diff
@@ -0,0 +1,149 @@
+<?php
```

Comment:
Hi, @ifandpromiss 

you should not simply copy the same code to the framework, because we make a wrapper framework to Joomla and will try to extend the functionality. The simple copied version can not be tracked to the newest code in the later Joomla versions.